src/main/java/hudson/plugins/testlink/AbstractTestLinkBuilder.java

```diff
@@ -182,13 +188,13 @@ public AbstractTestLinkBuilder(String testLinkName, String testProjectName, Stri
      * @deprecated
      */
     public AbstractTestLinkBuilder(String testLinkName, String testProjectName, String testPlanName,
-            String platformName, String buildName, String customFields, Boolean executionStatusNotRun,
+            String platformName, String buildName, String customFields, String testPlanCustomFields, Boolean executionStatusNotRun,
```

Comment:
The deprecated constructor can be left as is, and just pass null/empty values to the current constructor.